src/test/kotlin/com/pubnub/api/CommonUtils.kt

```diff
@@ -0,0 +1,101 @@
+package com.pubnub.api
+
+import com.github.tomakehurst.wiremock.client.WireMock
```

Comment:
There is CommonUtils.kt file located here:
src/integrationTest/kotlin/com/pubnub/api/CommonUtils.kt
that has very similar content. Having two of them is desired?

Reply:
It's necessary for the tests to compile. Those two sources can't share the code between them, and because I've removed the fixtures plugin we need to have the copy in both places. Eventually when we reduce the number of integrationTests we might not need it